strings/base_collections_base.h

```diff
@@ -1,5 +1,45 @@
 WINRT_EXPORT namespace winrt
 {
+    namespace impl
```

Comment:
Don't nest namespaces. This causes the impl namespace to be exported as part of a C++ module. Instead, start a new root namespace definition every time e.g. `namespace winrt::impl`